doc/manual/src/release-notes/rl-2.4.md

```diff
@@ -0,0 +1,13 @@
+# Release 2.4 (202X-XX-XX)
+
+  - When using the `nix` command, it is now deprecated to pass
+    top-level flags (e.g. `--option`) after the subcommand. For
+    example, `nix show-config --option foo bar` should instead be
+    written `nix --option foo bar show-config`.
```

Comment:
```suggestion
```

If this only affects plugin loading (which is undocumented anyway) then there is no need to call this deprecated since it works fine for every other setting.

Reply:
Deprecation removed